app/services/albums.js

```diff
@@ -16,29 +17,46 @@ exports.findAlbums = () => {
   });
 };
 
-exports.findPhotosByAlbumId = albumId => {
+exports.findAlbums = () => {
   const options = {
     method: 'GET',
-    uri: albums_api_url.photos,
-    qs: { albumId },
+    uri: albums_api_url.albums,
     json: true
   };
-  logger.info(`Consuming an external api with url: ${options.uri}?albumId=${albumId}`);
+  logger.info(`Consuming an external api with url: ${options.uri}`);
   return request(options).catch(err => {
     logger.error(err.message);
     throw errors.externalApiError('Error consuming external API');
   });
 };
 
-exports.findAlbumById = id => {
+exports.findPhotosByAlbumId = albumId => {
   const options = {
     method: 'GET',
-    uri: `${albums_api_url.albums}/${id}`,
+    uri: albums_api_url.photos,
+    qs: { albumId },
     json: true
   };
-  logger.info(`Consuming an external api with url: ${options.uri}`);
+  logger.info(`Consuming an external api with url: ${options.uri}?albumId=${albumId}`);
   return request(options).catch(err => {
     logger.error(err.message);
     throw errors.externalApiError('Error consuming external API');
   });
 };
+
+exports.findAlbumById = findAlbumById;
+
+exports.buyAlbum = (userId, albumId) =>
```

Comment:
check https://sequelize.org/master/manual/models-usage.html#-code-findorcreate--code----search-for-a-specific-element-or-create-it-if-not-available